tasks/nmg_transform.py

```diff
@@ -0,0 +1,47 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+class NgmTransForm(BaseSalesforceApiTask):
+    def _run_task(self):
+        def getId(line):
+            index1 = line.find('(') + 1
+            index2 = line.find("'", index1 + 1) + 1
+            return line[index1:index2]
+
+        def createInsertLine(id, networkId, count):
+            insertLine = "INSERT INTO \"NetworkMemberGroup\" VALUES("
```

Comment:
An fstring could clean this up nicely to make it clearer what the method intends to do.